website/src/routes/apiv1.js

```diff
@@ -1,173 +1,13 @@
 var express = require('express');
```

Comment:
It was deprecated but working, we can keep the message for 1-2 weeks and then remove it